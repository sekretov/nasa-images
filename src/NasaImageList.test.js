import React from 'react';
import ReactDOM from 'react-dom';
import NasaImageList from './NasaImageList';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<NasaImageList />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders and matches the previous snapshot', () => {
    const tree = renderer.create(<NasaImageList />).toJSON();
    expect(tree).toMatchSnapshot();
});

