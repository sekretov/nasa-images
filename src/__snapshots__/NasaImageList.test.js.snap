// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders and matches the previous snapshot 1`] = `
<ul>
  Loading...
</ul>
`;
